app/views/layouts/_navbar.haml

```diff
@@ -1,12 +1,28 @@
-%nav.navbar.navbar-expand-lg.mb-2.bg-light{aria: {label: 'Main'}}
-  - if @current_user
-    = navbar_link 'Passengers', passengers_path
-    = navbar_link 'Add New Passenger', new_passenger_path
-    = navbar_link 'Dispatch Log', log_index_path
-    = navbar_link 'Archived Passengers', archived_passengers_path
-  - if @current_user&.admin?
-    = navbar_link 'Manage Users', users_path
-    = navbar_link 'Manage Mobility Devices', mobility_devices_path
-  = navbar_link 'Site Feedback', feedback_index_path
-  = navbar_link 'Brochure', brochure_passengers_path
-  = link_to 'Logout', destroy_session_path, class: 'ml-auto mr-2 nav-link btn btn-secondary'
+%nav.navbar.navbar-expand-lg.navbar-light.bg-light{aria: {label: 'Main'}}
+  %ul.navbar-nav.mr-auto
+    - if @current_user
+      %li.nav-item.dropdown
+        = link_to 'Passengers', '#', class: 'nav-link dropdown-toggle', id: 'passenger-dropdown',
+          data: {toggle: 'dropdown'}, aria: {haspopup: 'true', expanded: 'false'}
+        .dropdown-menu{aria: { labelledby: 'passenger-dropdown' }}
+          = link_to 'Current', passengers_path, class: 'dropdown-item'
+          = link_to 'Archived', archived_passengers_path, class: 'dropdown-item'
+          = link_to pending_passengers_path, class: 'dropdown-item' do
+            Pending
+            %span.badge.badge-warning= Passenger.pending.count
```

Comment:
I think this badge will be a bit more effective if it isn't displayed when there are 0.